docs/site/src/demos/lists/SingleLineList.js

```diff
@@ -0,0 +1,222 @@
+// @flow weak
+
+import React, { Component } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import {
+  List,
+  ListItem,
+  ListItemAvatar,
+  ListItemIcon,
+  ListItemSecondaryAction,
+  ListItemText,
+} from 'material-ui/List';
+import Avatar from 'material-ui/Avatar';
+import FolderIcon from 'material-ui/svg-icons/folder';
+import IconButton from 'material-ui/IconButton';
+import { LabelCheckbox } from 'material-ui/Checkbox';
+import Layout from 'material-ui/Layout';
+import Text from 'material-ui/Text';
+import TriangleImageLight from 'docs/site/assets/images/bg-triangle-light.svg';
```

Comment:
You see me coming but simplicity might be just as good: removing that fancy effect.